charts/datadog/templates/NOTES.txt

```diff
@@ -337,7 +337,7 @@ You are using datadog.orchestratorExplorer.enabled but you disabled the cluster
 To enable it please set clusterAgent.enabled to 'true'.
 {{- end }}
 
-{{- if .Values.providers.gke.autopilot}}
+{{- if and (.Values.providers.gke.autopilot) (not .Values.datadog.envDict.DD_CI)}}
```

Comment:
The `.Values.datadog.envDict.DD_CI` is a workaround for CI tests running in non-autopilot clusters